src/data/code-samples/en/input-instructions/swift-ui.md

```diff
@@ -0,0 +1,33 @@
+# Input instructions - SwiftUI
+
+In SwiftUI, we recommend using a [`Text`](https://developer.apple.com/documentation/swiftui/text) or [`Label`](https://developer.apple.com/documentation/swiftui/label) views to show input instructions. The input instructions message should also be posted to assistive technologies by using an [`accessibility announcement`](../Techniques/accessibility-announcement.md).
+
+You could also use a third party library to displaying instructions. Unfortunately, accessibility is often not considered in the implementations.
```

Comment:
displaying --> display

Reply:
Maybe we should remove 3rd party reference for the `input` code samples, because with SwiftUI it's easier compared to UIKit to add instructions and show errors etc.